templates/auth/login.html

```diff
@@ -13,36 +13,44 @@
     </div>
   {% endif %}
 
-  <form class="card" style="border-radius: 2%" method="post">
-    {{ form.csrf_token }}
+  <div class="card" style="border-radius: 2%">
     <div class="card-body p-6">
       <h1 class="card-title">Welcome back!</h1>
-      <div class="form-group">
-        <label class="form-label">Email address</label>
-        {{ form.email(class="form-control", type="email", autofocus="true") }}
-        {{ render_field_errors(form.email) }}
-      </div>
-
-      <div class="form-group">
-        <label class="form-label">
-          Password
-        </label>
-        {{ form.password(class="form-control", type="password") }}
-        {{ render_field_errors(form.password) }}
-        <div class="text-muted">
-          <a href="{{ url_for('auth.forgot_password') }}" class="small">
-            I forgot my password
-          </a>
+      <form method="post">
+        {{ form.csrf_token }}
+
+        <div class="form-group">
+          <label class="form-label">Email address</label>
+          {{ form.email(class="form-control", type="email", autofocus="true") }}
+          {{ render_field_errors(form.email) }}
         </div>
-      </div>
 
-      <div class="form-footer">
-        <button type="submit" class="btn btn-primary btn-block">Log in</button>
-      </div>
+        <div class="form-group">
+          <label class="form-label">
+            Password
+          </label>
+          {{ form.password(class="form-control", type="password") }}
+          {{ render_field_errors(form.password) }}
+          <div class="text-muted">
+            <a href="{{ url_for('auth.forgot_password') }}" class="small">
+              I forgot my password
+            </a>
+          </div>
+        </div>
 
+        <div class="form-footer">
+          <button type="submit" class="btn btn-primary btn-block">Log in</button>
+        </div>
 
+      </form>
+      {% if connect_with_proton %}
```

Comment:
can we also add "Sign up with Proton" on the sign up page which works similarly to this?

Reply:
Button added. It calls to the exact same code, as from the oauth perspective the logic is the same in both the login and create account (if you try to log in with proton and you don't have a SimpleLogin account, one will be created for you).